source/style.css

```
@@ -209,41 +206,29 @@ footer {
 	height: 10vh;
 }
 
-div.fixedbutt {
-	background-color: rgb(53 53 53);
-	background-image: url("images/vertical_align.svg");
-	background-repeat: no-repeat;
-	background-position: center;
-	position: fixed;
-	z-index: 999;
-	top: 3vh;
-	right: 3vw;
-	border: 1px solid white;
+#controls-pane.open {
 
-	--padding: 5px;
-	padding: 3px;
-	padding-left: var(--padding);
-	padding-right: var(--padding);
-}
-
-div.fixedbutt > a {color: #FFFFFF00; text-decoration: none;}
-div.fixedbutt > a:visited {color: var(--color);}
-div.fixedbutt:hover {filter: brightness(1.8);}
-
-
-#controls-pane {
-	flex: 1 1 0;
 	max-width: 300px;
 	padding: 2rem 1rem;
 	background-color: #2E3047;
 	position: fixed;
 	top: 0;
-	left: 0;
 	height: 100vh;
 	z-index: 200;
 }
 
-#controls-pane h3 {
+#controls-pane.closed {
+	position: fixed;
+	max-width: 300px;
+	padding: 2rem 1rem;
+	background-color: #115204;
+	top: 0;
+	left: -300px;
+	height: 100vh;
+	z-index: 200;
+}
+
+h3#controls-pane {
```

```suggestion
#controls-pane h3 {
```
As far as I can tell, we don't have an <h3> with id `controls-pane` but we do have a `controls-pane` with an <h3> inside :)
Maybe the plan was to delete these styles?

---

I fixed this for you in #27